aggregations.mdx

```diff
@@ -84,22 +84,43 @@ class User:
     )
 ```
 
-Because this code shows a `bucket_duration` of 1 day, Chalk will aggregate transaction data into 1 day buckets. Chalk
-will then use these buckets to serve `total_transaction_amount` for the past 30 days and 90 days. As new data arrives,
-the relevant bucket is modified to include the data.
+Because this code shows a `bucket_duration` of 1 day, Chalk will aggregate transaction data into 1 day buckets of
+timeseries data. Chalk will then use this timeseries data to serve `total_transaction_amount` for the past 30 days and
+90 days. As new data arrives, the relevant timeseries buckets are modified to include the data. Over time, the oldest
+buckets are removed from your online store once they cannot be used by any time window.
+
+The number of buckets is determined by your longest time window divided by your bucket duration. For example, if your
+time window is 90 days and your bucket duration is 1 day, you would have 90 buckets. If your bucket duration is set to 1
+minute, you would instead have 129,600 buckets.
+
+Buckets are aligned starting from the Unix epoch, ignoring leap seconds. To serve windowed feature queries, Chalk uses
+all buckets containing any overlap with the requested time window.
+
+### Managing aggregations
+
+Use [`chalk aggregate backfill`](/cli/aggregate_backfill) to backfill aggregation for a windowed feature. This command
+is useful if you change your feature's time windows or `bucket_duration` values.
+
+To view existing aggregations, use [`chalk aggregate list`](/cli/aggregate_list).
+
+Each of these commands outputs a table of your aggregations:
+
+```
+ Series  Namespace    Group                Agg     Bucket  Retention  Aggregation  Dependent Features
+────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+ 1       transaction  user_id merchant_id  amount  1d      30d        sum          user.txn_sum_by_merchant merchant.txn_sum_by_user
+ 1       transaction  user_id merchant_id  amount  1d      30d        count        user.txn_count_by_merchant
+ 2       transaction  user_id              amount  1d      30d        sum          user.txn_sum
+```
 
-### How aggregation works
```

Comment:
This section was pretty repetitive given the paragraphs above it, so I removed it